### ISSUE TITLE
livequery support

### ISSUE THREAD

**Issue by julien9999** [ramonthegreat/ios-parse-64bit-support-titanium-module#1]
Hi, is livequery supported in this version ?
thanks in advance

**Comment by ramonthegreat** [comment]
No. This does not support LiveQuery, but you may modify the source code and add the parse live query feature.

**Comment by julien9999** [comment]
I am willing to pay for this feature, could you do it ?

**Comment by julien9999** [comment]
@ramonthegreat can you help?

**Comment by ramonthegreat** [comment]
@julien9999 Yes. Please send me an email at clockpensolution@gmail.com
